cocos2d/videoplayer/video-player-impl.js

```diff
@@ -158,23 +158,24 @@ let VideoPlayerImpl = cc.Class({
         video.style.height = height + 'px';
     },
 
-    _createDom () {
+    _createDom (mute, volume) {
         let video = document.createElement('video');
         video.style.position = "absolute";
         video.style.bottom = "0px";
         video.style.left = "0px";
         video.className = "cocosVideo";
         video.setAttribute('preload', 'auto');
         video.setAttribute('webkit-playsinline', '');
-        // This x5-playsinline tag must be added, otherwise the play, pause events will only fire once, in the qq browser.
-        video.setAttribute("x5-playsinline", '');
```

Comment:
Why remove the attribute for X5?

Reply:
I'm not sure if I first pulled another version of the engine or if I blindly deleted this. This was not the purpose and it is added back now.